### PR TITLE
Improve dev server (again)

### PR DESCRIPTION
**Pull request by TymanWasTaken**
Why did 6pak feel the need to force push

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "build": "rollup -c --configPlugin @rollup/plugin-typescript",
     "dev": "pnpm run build -w",
-    "deploy": "adb shell am start -S -n com.discord/.MainActivity",
+    "deploy": "pnpm run build && adb push dist/Aliucord.js.bundle /sdcard/AliucordRN/Aliucord.js.bundle && adb shell am start -S -n com.discord/.MainActivity",
```
Comment (Vendicated) [inline package.json:12]: && does not work on windows

Comment (C10udburst) [reply package.json:12]: wdym? it does

Comment (TymanWasTaken) [reply package.json:12]: ~~yarn makes it work on windows~~ how else are you supposed to do that then

**Review by Vendicated — APPROVED** [review]

**Comment by Vendicated** [comment]
can u please rebase

**Comment by TymanWasTaken** [comment]
New pr #54

**Pull request closed without merging**